selfhost/parser.jakt

```diff
@@ -1389,6 +1407,9 @@ struct Parser {
 
         let lhs = .parse_operand()
         expr_stack.push(lhs)
+        if lhs is Garbage {
+            .error("Unsupported expression", lhs.span())
+        }
```

Comment:
I think we may just want to give an error in `parse_operand_base` when we first create the garbage rather than detecting Garbage later and giving an error then.

Reply:
Yeah, I agree it makes more sense. Fixed.
Maybe the error could bubble up and be handled more gracefully by the caller, who then could provide more information?